packages/domains-table/src/domains-table/domains-table-ssl-cell.tsx

```diff
@@ -40,13 +41,15 @@ export default function DomainsTableSslCell( {
 
 	if ( sslStatus ) {
 		button = (
-			<a
+			<button
 				className="domains-table-row__ssl-status-button"
-				href={ `${ domainManagementLink }?ssl-open=true` }
-				onClick={ ( event ) => event.stopPropagation() }
+				onClick={ ( event ) => {
+					event.stopPropagation();
+					navigate( `${ domainManagementLink }?ssl-open=true` );
```

Comment:
```suggestion
					page.show( `${ domainManagementLink }?ssl-open=true` );
```

Then we can use page.show instead of navigate function